packages/one-app-dev-bundler/esbuild/generateESBuildOptions.js

```diff
@@ -156,6 +156,7 @@ const generateESBuildOptions = async ({ watch, useLiveReload }) => {
   }
 
   return {
+    commonConfig,
```

Comment:
Avoid exporting the common config, it should be considered private to this file.

Instead, export a new value here 'externalsConfig' which derives from the commonConfig. (See me previous comment about this too)